### PR TITLE
Fixed german translation of README.md

### PR DESCRIPTION
**Pull request by Julian-Sz**
Fixed many typos etc

### CODE REVIEW

File: docs/README_DE.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 [English](../README.md) | [한국어](README_KO.md) | [Español](README_ES.md) | [Русский](README_RU.md) | [简体中文](README_CN.md) | [Français](README_FR.md) | Deutsch
 
-OpenTabletDriver ist ein Open-Source, platformübergreifender, Benutzermodus Tablet Treiber. Das Ziel von OpenTabletDriver ist es so Cross-Platform wie möglich zu sein mit der höchsten Kompatibilität und mit einer einfach zu konfigurierenden grafischen Benutzeroberfläche.
+OpenTabletDriver ist ein plattformübergreifender Open Source Benutzermodustreiber für Grafiktabletts. Das Ziel von OpenTabletDriver ist es, so Cross-Plattform wie möglich zu sein, mit der höchsten Kompatibilität und mit einer einfach zu konfigurierenden grafischen Benutzeroberfläche.
```
Comment (X9VoiD) [inline docs/README_DE.md:7]:
```suggestion
OpenTabletDriver ist ein plattformübergreifender Open Source Benutzermodustreiber für Grafiktabletts. Das Ziel von OpenTabletDriver ist es, so Cross-Platform wie möglich zu sein, mit der höchsten Kompatibilität und mit einer einfach zu konfigurierenden grafischen Benutzeroberfläche.
```

**Review by X9VoiD — CHANGES_REQUESTED** [review]

**Review by X9VoiD — APPROVED** [review]

**Comment by C0D3-M4513R** [comment]
I'd either remove or finish the last sentence. Not a blocker in my eyes though.
This pr greatly improves readability (something that was severely lacking in #1777 ).

**Review by C0D3-M4513R — APPROVED** [review]

**Comment by C0D3-M4513R** [comment]
wWe should add localization and documentation tags

**Comment by InfinityGhost** [comment]
Is this good to go?

**Comment by Julian-Sz** [comment]
Yes, ready to merge

**Comment by jamesbt365** [comment]
somebody merge this lmao

**Pull request merged**